src/utils/todo/task-filing/content-loader.ts

```diff
@@ -140,11 +141,10 @@ export const addSubtitleOrContent = (todoItem: TodoItemI): void => {
       } else if (todoItem.payErrorObj) {
         todoItem.subtitle = t('text.todoItem.status.paymentIncomplete')
       } else if (filingWithNR.includes(todoItem.name)) {
-        // TO-DO: the subtitle is special if nameRequest exists --- updated this when NameRequest store is implemented
-        // if (this.getNameRequest) {
-        //   subtitle = `NR APPROVED - ${this.expiresText(this.getNameRequest)}`
-        // } else { subtitle = 'DRAFT' }
         todoItem.subtitle = t('text.todoItem.status.draft')
+        if (todoItem.nameRequest) {
+          todoItem.subtitle = nrSubtitle(todoItem.nameRequest)
+        }
```

Comment:
This can be simplified as: `todoItem.subtitle = todoItem.nameRequest ? nrSubtitle(todoItem.nameRequest) : t('text.todoItem.status.draft')`